entries/l/laybuy.com.json

```diff
@@ -3,7 +3,8 @@
     "domain": "laybuy.com",
     "contact": {
       "facebook": "PaybyLaybuy",
-      "twitter": "PaybyLaybuy"
+      "twitter": "PaybyLaybuy",
+      "form": "https://help.laybuy.com/hc/en-us/requests/new?ticket_form_id=360004967093"
```

Comment:
```suggestion
      "form": "https://help.laybuy.com/hc/en-us/requests/new"
```

Reply:
I think it's more beneficial to be taken directly to the recommended option to submit the 2FA request/suggestion. Plus depending on if the company's customer service team does reporting or not, having all 2FA requests fall under the same category would be more useful.